File: app/src/main/java/com/even/recyclerviewdemo/beans/ClassifyBean.java
package com.even.recyclerviewdemo.beans;

import com.even.commonrv.adapter.BaseViewHolder;
import com.even.commonrv.bean.BaseListPagerBean;
import com.even.recyclerviewdemo.R;

/**
 * @author Created by Even on 2019/2/21
 * Emial: emailtopan@163.com
 * 分类
 */
public class ClassifyBean extends BaseListPagerBean {

    public ClassifyBean(String textLeft, String textRight) {
        super(textLeft, textRight);
    }

    @Override
    public void cover(BaseViewHolder holder, BaseListPagerBean bean) {
        holder.setText(R.id.tvLeft, textLeft);
    }

    @Override
    public int getContentViewId() {
        return R.layout.item_classify;
    }
}

File: app/src/main/java/com/even/recyclerviewdemo/beans/ClassifyBean.kt
package com.even.recyclerviewdemo.beans

import android.content.Context
import com.even.commonrv.adapter.BaseViewHolder
import com.even.commonrv.bean.BaseListPagerBean
import com.even.recyclerviewdemo.R

/**
 * @author Created by Even on 2019/2/21
 * Email: emailtopan@163.com
 * 分类
 */
class ClassifyBean(var textLeft: String) : BaseListPagerBean() {
    override fun cover(
        context: Context,
        holder: BaseViewHolder,
        position: Int
    ) {
        holder.setText(R.id.tvLeft, textLeft)
    }

    override val contentViewId: Int = R.layout.item_classify
}